src/librustc_target/spec/mod.rs

```diff
@@ -397,6 +397,7 @@ supported_targets! {
     ("thumbv7m-none-eabi", thumbv7m_none_eabi),
     ("thumbv7em-none-eabi", thumbv7em_none_eabi),
     ("thumbv7em-none-eabihf", thumbv7em_none_eabihf),
+    ("thumbv8m-none-eabi", thumbv8m_none_eabi),
```

Comment:
`thumbv8m.base-none-eabi` would be better to distinguish from a `thumbv8m.main` target in the future.